src/cluster.py

```diff
@@ -25,57 +26,39 @@
 PATRONI_SERVICE = "patroni"
 
 
-class ClusterAlreadyRunningError(Exception):
-    """Exception raised when there is already a running cluster."""
+class Patroni:
+    """This class handles the bootstrap of a PostgreSQL database through Patroni."""
 
     pass
```

Comment:
This line is unnecessary.

Reply:
Good catch. Removed `pass` on https://github.com/canonical/postgresql-operator/pull/8/commits/d2d0915c789894145f29c296de6b4ec5492cf382.